.travis.yml

```
@@ -25,18 +25,16 @@ script:
 
 before_deploy:
   # decrypt ReTest secret key
-  - openssl aes-256-cbc -K $encrypted_58a7d4f10478_key -iv $encrypted_58a7d4f10478_iv
-      -in retest-gmbh-gpg.asc.enc -out retest-gmbh-gpg.asc -d
+  - openssl aes-256-cbc -K "$encrypted_58a7d4f10478_key" -iv "$encrypted_58a7d4f10478_iv"
+    -in retest-gmbh-gpg.asc.enc -out retest-gmbh-gpg.asc -d
```

This needs two more spaces indentation:

```suggestion
      -in retest-gmbh-gpg.asc.enc -out retest-gmbh-gpg.asc -d
```